opencti-platform/opencti-graphql/src/types/work.d.ts

```diff
@@ -16,7 +16,7 @@ interface Work {
   received_time: string | null
   processed_time: string | null
   completed_time: string | null
-  completed_number: string
```

Comment:
I don't know why it was set as string, it's used as number in work domain
https://github.com/OpenCTI-Platform/opencti/blob/839fca43230f54fd4c71d08e03410ba151be2ea8/opencti-platform/opencti-graphql/src/domain/work.js#L193